composer.json

```diff
@@ -1,31 +1,25 @@
 {
-  "name":        "tijsverkoyen/css-to-inline-styles",
-  "type":        "library",
+  "name": "tijsverkoyen/css-to-inline-styles",
+  "type": "library",
   "description": "CssToInlineStyles is a class that enables you to convert HTML-pages/files into HTML-pages/files with inline styles. This is very useful when you're sending emails.",
-  "homepage":    "https://github.com/tijsverkoyen/CssToInlineStyles",
-  "license":     "BSD",
-  "authors":     [
+  "homepage": "https://github.com/tijsverkoyen/CssToInlineStyles",
+  "license": "BSD-3-Clause",
+  "authors": [
     {
-      "name":  "Tijs Verkoyen",
+      "name": "Tijs Verkoyen",
       "email": "css_to_inline_styles@verkoyen.eu",
-      "role":  "Developer"
+      "role": "Developer"
     }
   ],
-  "require":     {
-    "php":                  ">=5.3.0",
```

Comment:
I removed it, because in the real world there are a lot of outdated servers. I won't support issues with those older version, but I don't want to force them also.